static/cloudformation/kubernetes.template

```diff
@@ -0,0 +1,44 @@
+Parameters:
+  trustedAccount:
+    Type: String
+    Description: The id of the roadie account to allow access.
+  externalId:
+    Type: String
+    Description: The external Id.
```

Comment:
Same here

Reply:
I am not sure whether this one should be passed in yet.